packs/data/equipment.db/ring-of-torag.json

```diff
@@ -35,7 +35,34 @@
             }
         },
         "quantity": 1,
-        "rules": [],
+        "rules": [
+            {
+                "domain": "all",
+                "key": "RollOption",
+                "option": "ring-of-torag",
+                "toggleable": true
+            },
```

Comment:
Thinking about it this one might do well with a label, "Defending against a fire spell" or something along those lines. I know shark likes things to be more descriptive.